src/shared/features/AppletSettings/TransferOwnershipSetting/TransferOwnershipSetting.tsx

```diff
@@ -1,39 +1,60 @@
 import { useEffect, useRef, useState } from 'react';
-import { useTranslation } from 'react-i18next';
+import { Trans, useTranslation } from 'react-i18next';
 import { Box } from '@mui/material';
 
-import { applet } from 'shared/state';
+import { applet, banners } from 'shared/state';
 import { TransferOwnership } from 'modules/Dashboard/features/Applet/TransferOwnership';
-import { SuccessTransferOwnershipPopup } from 'modules/Dashboard/features/Applet/Popups';
 import { TransferOwnershipRef } from 'modules/Dashboard/features/Applet/TransferOwnership/TransferOwnership.types';
 import { Mixpanel } from 'shared/utils';
+import { useAppDispatch } from 'redux/store';
 
 import { StyledTransferOwnershipForm } from './TransferOwnershipSetting.styles';
 import { StyledAppletSettingsButton } from '../AppletSettings.styles';
 
 export const TransferOwnershipSetting = () => {
   const { t } = useTranslation('app');
+  const dispatch = useAppDispatch();
   const { result: appletData } = applet.useAppletData() ?? {};
 
   const [isSubmitted, setIsSubmitted] = useState(false);
-  const [emailTransfered, setEmailTransfered] = useState('');
-  const [transferOwnershipPopupVisible, setTransferOwnershipPopupVisible] = useState(false);
+  const [emailTransferred, setEmailTransferred] = useState('');
   const transferOwnershipRef = useRef<TransferOwnershipRef | null>(null);
 
   const dataTestid = 'applet-settings-transfer-ownership';
 
-  const handleSuccessPopupClose = () => {
-    setTransferOwnershipPopupVisible(false);
-    setEmailTransfered('');
+  const handleSuccess = () => {
+    setEmailTransferred('');
     transferOwnershipRef.current?.resetEmail();
   };
 
   useEffect(() => {
-    if (!emailTransfered) return;
+    if (!emailTransferred) return;
```

Comment:
if it'll not take a long time let's get rid of `useEffect` that could be replaced with a callback as I guess into the places where triggered `setEmailTransferred(true)`